base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
I had made that suggestions as well, but Jeff thinks it's a little long. I had suggested `chain`, but that might conflict with the iterator of the same name.